src/Components/WebAssembly/testassets/HostedInAspNet.Server/Startup.cs

```diff
@@ -39,14 +49,31 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, BootReso
                 app.UseWebAssemblyDebugging();
             }
 
-            app.UseBlazorFrameworkFiles();
+            if (mapAllApps || mapAlternativePathApp)
+            {
+                app.UseBlazorFrameworkFiles("/app");
+            }
+
+            if(!mapAlternativePathApp || mapAllApps) 
```

Comment:
Mainly to fix whitespace:

```suggestion
            if (mapAllApps || !mapAlternativePathApp)
```